test/tests/api/v2_0/os_install_tests.py

```diff
@@ -46,8 +50,75 @@ def teardown(self):
     def __get_data(self):
         return loads(self.__client.last_response.data)
 
+    def __get_compute_nodes(self):
+        Api().nodes_get_all()
```

Comment:
I am not sure whether the on-http python client support query, because `/api/2.0/nodes?type=compute` will directly return all compute nodes, so you don't have to do filter by yourself.

Anyway, your current code is still OK.